paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentOptionsItem.kt

```diff
@@ -0,0 +1,58 @@
+package com.stripe.android.paymentsheet
+
+import android.content.res.Resources
+import com.stripe.android.model.PaymentMethod
+
+internal sealed class PaymentOptionsItem {
```

Comment:
This used to be `PaymentOptionsAdapter.Item`. No code changes, just renamed and moved.